modules/nodes-nodes-working-evacuating.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-nodes-managing.adoc
+
+[id='nodes-nodes-managing-evacuating_{context}']
+= Evacuating pods on nodes
+
+Evacuating pods allows you to migrate all or selected pods from a given node or
+nodes. 
+
+You can only evacuate pods backed by a replication controller. The replication controllers create new pods on
+other nodes and removes the existing pods from the specified node(s). 
+
+Bare pods, meaning those not backed by a replication controller, are unaffected by default.
+You can evacuate a subset of pods by specifying a pod-selector. Pod selectors are
+based on labels, so all the pods with the specified label will be evacuated.
+
+.Prerequisites
+
+* Nodes must first be marked unschedulable to perform pod evacuation.
```

Comment:
give the example `oc adm cordon` command here

Reply:
seems we are also using `oc adm manage-node <node> --schedulable=false`.  not sure what the difference is or which is preferred.